### ISSUE TITLE
PT007 doesn't seem right

### ISSUE THREAD

**Issue by webknjaz** [m-burst/flake8-pytest-style#2]
### Description

`@pytest.mark.parametrize` should check for iterable containers rather than enforce list.
I, for instance, prefer using immutable types, like tuples so I never use lists there. Another valid case would be using sets, which may prove to be useful if one wants to ensure there's no duplicates.

### What I Did

```python
@pytest.mark.parametrize(
	'adapter_type',
	(
		'builtin',
		'pyopenssl',
	),
)
```
/
```python
@pytest.mark.parametrize(
	'is_trusted_cert,tls_client_identity',
	(
		(True, 'localhost'), (True, '127.0.0.1'),
		(True, '*.localhost'), (True, 'not_localhost'),
		(False, 'localhost'),
	),
)
```

**Comment by m-burst** [comment]
Hi @webknjaz,

My intention for PT006 and PT007 was primarily to enforce consistency between all tests in a single codebase (i.e. to disallow having a list in one test and a tuple in another).
I understand that the rules are opinionated, but at the moment of writing the rules I wanted to keep them simple, so I chose to enforce the types that my team at work agreed upon.
I might make the rules configurable in the future if I have time (or I would gladly accept a PR).

**Comment by webknjaz** [comment]
Sounds good. It's just that consistency style can be project-specific so it'd be nice to be able to select that style.

**Comment by m-burst** [comment]
Hi @webknjaz,
I've just released v0.6.0 in which you can configure the expected types for `PT006` and `PT007`.
You can find the details in the [changelog](https://github.com/m-burst/flake8-pytest-style#change-log).

**Comment by sergiowalls** [comment]
Hey, sorry for reopening this again... I'm wondering why this is not right

```python3
@pytest.mark.parametrize("date_range, expected", [
    (("2019-01-20", "2019-01-20"), ["2019-01-20"]),
    (("2019-01-20", "2019-01-21"), ["2019-01-20", "2019-01-21"]),
    (("2019-01-20", "2019-01-22"), ["2019-01-20", "2019-01-21", "2019-01-22"]),
    (("2020-02-28", "2020-03-01"), ["2020-02-28", "2020-02-29", "2020-03-01"])
])
def test_get_date_range(date_range, expected):
    start, end = date_range

    assert get_date_range(start, end) == expected
```

I can see that the input and the output are different (tuple and list), but I don't get the problem with that.

**Comment by m-burst** [comment]
Hi @sergiowalls,
Please post the exact error that you are getting, as well as your flake8 settings related to flake8-pytest-style (preferably in a new issue).
